### PR TITLE
Add more detail to the failure notice so that we can actually see what is going on.

### PR DESCRIPTION
**Pull request by brendandburns**
@roberthbailey @zmerlynn @quinton-hoole

### CODE REVIEW

**Comment by roberthbailey** [comment]
Is this test currently failing?

File: test/e2e/certs.go [hunk 1]
```diff
@@ -41,8 +41,8 @@ var _ = Describe("MasterCerts", func() {
 			cmd := exec.Command("gcloud", "compute", "ssh", "--project", testContext.CloudConfig.ProjectID,
 				"--zone", testContext.CloudConfig.Zone, testContext.CloudConfig.MasterName,
 				"--command", fmt.Sprintf("ls /srv/kubernetes/%s", certFile))
-			if _, err := cmd.CombinedOutput(); err != nil {
-				Fail(fmt.Sprintf("Error checking for cert file %s on master: %v", certFile, err))
+			if output, err := cmd.CombinedOutput(); err != nil {
+				Fail(fmt.Sprintf("Error checking for cert file %s on master: %v\n%s", certFile, err, string(output)))
```
Comment (roberthbailey) [inline test/e2e/certs.go]: You should use Failf now and get rid of the fmt.Sprintf

**Comment by zmerlynn** [comment]
Yeah, also curious why this was opened, but other than @roberthbailey's comment LGTM

File: test/e2e/certs.go (hunk 1 above)
Comment (brendandburns) [inline test/e2e/certs.go]: done.

**Comment by brendandburns** [comment]
It flaked once overnight in either GKE or GCE (can't remember which off the top of my head) and the output was useless to see why.

**Comment by roberthbailey** [comment]
LGTM. Will merge on green.

**Comment by roberthbailey** [comment]
Feel free to self-merge (or have @davidopp do it) once travis is happy since I'll be out this afternoon.

**Comment by quinton-hoole** [comment]
FYI, it flaked on kubernetes-e2e-gke-ci on runs 3657 and 3762

**Comment by brendandburns** [comment]
Shippable is a flake, I'm merging.

**Pull request merged**